internal/rbd/rbd_util.go

```diff
@@ -979,11 +921,36 @@ func (rv *rbdVolume) getImageInfo() error {
 		return err
 	}
 	rv.imageFeatureSet = librbd.FeatureSet(features)
-	err = rv.updateVolWithImageInfo(rv.conn.Creds)
+
+	// Get parent information.
+	// TODO: Replace GetParentInfo() after
+	// https://github.com/ceph/go-ceph/issues/347 is fixed.
```

Comment:
So, the current implementation of GetParentinfo() is not good because of the reaons mentioned in that go-ceph issue.
Plan is to add a new function, once we have that (in go-ceph v0.6) we can replace it with that.